docs/genome_to_collection.md

```diff
@@ -0,0 +1,235 @@
+    --tool $tool \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_ver \
+    --load_ver $load_ver \
+    --source_file_ext $source_file_ext \
+    --submit_job
+```
+
+# Step 4: Parse and Load Tool Outputs
+
+## Parse tool results
+
+### Parse tool computation results
+Parsing tool results for extracting insights from the processed data. This step involves parsing tool computation 
+results and genome taxa count information.
+
+```commandline
+# Parse tool computation results
+PYTHONPATH=. python src/loaders/genome_collection/parse_tool_results.py \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_ver \
+    --load_ver $load_ver \
+    --env $env
+	
+# Parse genome taxa count information
+attri_file=${kbase_collection}_${load_ver}_checkm2_gtdb_tk_kbcoll_genome_attribs.jsonl
+PYTHONPATH=. python src/loaders/genome_collection/compute_genome_taxa_count.py \
+    ../import_files/$env/$kbase_collection/$load_ver/$attri_file \
+    --load_ver $load_ver \
+    --kbase_collection $kbase_collection \
+    --env $env \
+    --input_source genome_attributes
+```
+
+## Load parsed results to ArangoDB
+
+```commandline
+# set up an SSH tunnel (Not required when using an internal KBase machine such as dev03) 
+USER_NAME=user_name                 # user name for login1.berkeley.kbase.us
+FORWARD=localhost:48000
+ssh -f -N -L $FORWARD:10.58.1.211:8531 \
+ $USER_NAME@login1.berkeley.kbase.us
+
+# execute arangoimport (Please ask system admin for the arangoDB credentials)
+PARSED_FILE=json_file_path          # the file path generated by the parsing script
+ARANGO_USER=arango_username         # arangoDB user name
+ARANGO_PW=arango_password           # arangoDB password
+ARANGO_DB=collections_dev           # arangoDB database name
+ARANGO_COLL=kbcoll_genome_attribs   # arangoDB collection name
+
+arangoimport --file $PARSED_FILE \
+    --server.endpoint tcp://$FORWARD \
+    --server.username $ARANGO_USER \
+    --server.password $ARANGO_PW \
+    --server.database $ARANGO_DB \
+    --collection $ARANGO_COLL \
+    --on-duplicate update
+```
+
+# Step 5: Create and Active the Collection
+
+## 5.1: Verify Permissions
+Before proceeding to create and activate the collection, execute the following command to verify if you have the 
+required permissions:
+
+```
+# Set the server URL and associated token
+SERVER='https://ci.kbase.us/services/collections'
+TOKEN='your_token_here' 
+
+curl -X 'GET' \
+"$SERVER/whoami/" \
+-H 'accept: application/json' \
+-H "Authorization: Bearer $TOKEN"
+```
+Example response:
+```
+{"user":"tgu2","is_service_admin":true}
+```
+Ensure that the "is_service_admin" field is set to true in the response. If it's false or if you encounter any errors, 
+contact system administrator to grant you the necessary permissions.
+
+## 5.2: Create (Save) a New Collection
+Utilize the `Save Collection` endpoint to create a new collection. Execute the following command to create a new collection:
```

Comment:
Do we want to direct users to the open api UI rather than, or in addition to, curl?

Reply:
sure. UI makes more sense.

Reply:
:+1: